components/src/Toggle/Toggle.js

```diff
@@ -16,8 +17,8 @@ const Slider = styled.span([], ({ toggled, disabled }) => ({
   right: "0",
   bottom: "0",
   border: "4px solid",
-  borderColor: getFill(disabled),
-  backgroundColor: (toggled ? getFill(disabled) : theme.colors.white),
+  borderColor: getOutlineColour(disabled),
+  backgroundColor: (toggled ? getOutlineColour(disabled) : getFillColour(disabled)),
```

Comment:
It doesn't read well when I see "use the border colour for the background color when toggled".

The same cleanup as `inputStyles` would help here as well.
There would be a duplicate color, but it would just be for backgroundColor.